sdks/python/apache_beam/runners/interactive/utils_test.py

```diff
@@ -327,5 +345,29 @@ def test_create_var_in_main(self):
     self.assertIs(getattr(main_session, name, None), value)
 
 
+@patch(
+    'apache_beam.io.gcp.internal.clients.storage.StorageV1',
+    return_value=MockStorageClient())
+class GCSUtilsTest(unittest.TestCase):
+  @patch(
+      'apache_beam.io.gcp.internal.clients.storage.StorageBucketsGetRequest',
+      return_value='test-bucket-not-found')
+  def test_assert_bucket_exists_not_found(self, mock_client, mock):
+    with self.assertRaises(ValueError):
+      utils.assert_bucket_exists('')
+
+  @patch(
+      'apache_beam.io.gcp.internal.clients.storage.StorageBucketsGetRequest',
+      return_value='test-bucket-not-verified')
```

Comment:
Can we add a test to verify if the warning is logged? See https://stackoverflow.com/questions/7472863/pydev-unittesting-how-to-capture-text-logged-to-a-logging-logger-in-captured-o for `self.assertLogs`.

You need to get the logger with the module name.

Reply:
Yes, I've included some additional code to check for the logger warning now.